synthtool/gcp/templates/node_library/CONTRIBUTING.md

```diff
@@ -39,6 +39,16 @@ accept your pull requests.
 
 ## Running the tests
 
+1.  To run the tests locally, you will need a Google Cloud Project with a [service account](https://cloud.google.com/docs/authentication/production#cloud-console) and an [API key](https://cloud.google.com/docs/authentication/api-keys)
+
+1.  Set the required environment variables. 
+```
+export GCLOUD_TESTS_PROJECT_ID=YOUR_GCP_PROJECT_ID
+export GCLOUD_TESTS_KEY=PATH_TO_YOUR_SERVICE_ACCOUNT_KEY.json
+export GCLOUD_TESTS_PROJECT_NUMBER=YOUR_GCP_PROJECT_NUMBER
```

Comment:
for most unit tests I think we only require that folks set a:

`GCLOUD_PROJECT` and a `GOOGLE_APPLICATION_CREDENTIALS`  (I think there are a few projects that require project number, but they're outliers).

Reply:
`we could then point out that tests sometimes take additional environment variables, which should be documented at the top of each test file.`
That's a great convention to follow. I don't think it's currently the majority case (at least for logging/errorreporting). Do you have a sample repo that already does this, so I can follow the formatting?

Reply:
I noticed this approach being used in `automl`:

https://github.com/googleapis/nodejs-automl/blob/master/samples/test/batch_predict.test.js#L23

Which has quite a few environment variables different than other libraries. Most libraries once a library is turned on on the account, just need the `GCLOUD_PROJECT` + `GOOGLE_APPLICATION_CREDENTIALS`.